src/widgets/helper/ChannelView.cpp

```diff
@@ -1124,9 +1124,11 @@ MessageElementFlags ChannelView::getFlags() const
     if (this->sourceChannel_ == app->twitch->mentionsChannel)
         flags.set(MessageElementFlag::ChannelName);
 
-    if (this->context_ == Context::ReplyThread)
+    if (this->context_ == Context::ReplyThread ||
+        !getSettings()->showReplyContext)
```

Comment:
Same thing here - I think this is more readable

```suggestion
    if (this->context_ == Context::ReplyThread &&
        getSettings()->showReplyContext)
```

Reply:
Same here: `!(this->context_ != Context::ReplyThread && getSettings()->showReplyContext)`. Though in this case I think this is less readable.